cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -397,6 +435,69 @@ endif()
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+# ----------------------------------------------------------------------
+# Google double-conversion
+
+macro(build_double_conversion)
+  set(DOUBLE_CONVERSION_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/double-conversion_ep/src/double-conversion_ep")
+  set(DOUBLE_CONVERSION_HOME "${DOUBLE_CONVERSION_PREFIX}")
+  set(double-conversion_INCLUDE_DIRS "${DOUBLE_CONVERSION_PREFIX}/include")
+  set(DOUBLE_CONVERSION_STATIC_LIB "${DOUBLE_CONVERSION_PREFIX}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}double-conversion${CMAKE_STATIC_LIBRARY_SUFFIX}")
+
+  set(DOUBLE_CONVERSION_CMAKE_ARGS
+    ${EP_COMMON_CMAKE_ARGS}
+    "-DCMAKE_INSTALL_PREFIX=${DOUBLE_CONVERSION_PREFIX}")
+
+  ExternalProject_Add(double-conversion_ep
+    ${EP_LOG_OPTIONS}
+    INSTALL_DIR ${DOUBLE_CONVERSION_PREFIX}
+    URL ${DOUBLE_CONVERSION_SOURCE_URL}
+    CMAKE_ARGS ${DOUBLE_CONVERSION_CMAKE_ARGS}
+    BUILD_BYPRODUCTS "${DOUBLE_CONVERSION_STATIC_LIB}")
+
+  add_library(double-conversion STATIC IMPORTED)
+  set_target_properties(double-conversion
+    PROPERTIES IMPORTED_LOCATION "${DOUBLE_CONVERSION_STATIC_LIB}")
+  add_dependencies(toolchain double-conversion_ep)
+  add_dependencies(double-conversion double-conversion_ep)
+  set(double-conversion_LIBRARIES double-conversion)
```

Comment:
`DOUBLE_CONVERSION_LIBRARIES` will be better for consistency.

Reply:
I used `double-conversion_LIBRARIES` as this is what double-conversion's CMake targets provides.

Reply:
(Ah, `*_LIBRARY` is better than`*_LIBRARIES`.)
`*_LIBRARY` is our rule for specifying the correct target. So I think that we can use our naming rule (`XXX_LIBRARY`) instead of double-conversion's naming rule here.
But this is a trivial. I don't care with `double-conversion_LIBRARY`.